tests/test_contact_specific_functions.py

```diff
@@ -13,54 +14,97 @@
 )
 
 
+def mock_rod_init(self):
+
+    "Initializing Rod"
+
+    """This is a small rod with 2 elements;
+    Initial Parameters:
+    radius = 1, length = 0.5,
+    tangent vector for both elements is (1, 0, 0),
+    stationary rod i.e velocity vector of each node is (0, 0, 0),
+    internal/external forces vectors are also (0, 0, 0)"""
+
+    self.n_elems = 2
+    self.position_collection = np.array([[1, 2], [0, 0], [0, 0]])
+    self.radius_collection = np.array([1, 1])
+    self.length_collection = np.array([0.5, 0.5])
+    self.tangent_collection = np.array([[1.0, 1.0], [0.0, 0.0], [0.0, 0.0]])
+    self.velocity_collection = np.array([[0.0, 0.0], [0.0, 0.0], [0.0, 0.0]])
+    self.internal_forces = np.array([[0.0, 0.0], [0.0, 0.0], [0.0, 0.0]])
+    self.external_forces = np.array([[0.0, 0.0], [0.0, 0.0], [0.0, 0.0]])
+
+
+def mock_rigid_body_init(self):
+
+    "Initializing Rigid Body as a cylinder"
+
+    """This is a rigid body cylinder;,
+    Initial Parameters:
+    radius = 1, length = 2,
+    center positioned at origin i.e (0, 0, 0),
+    cylinder's upright in x,y,z plane thus the director array,
+    stationary cylinder i.e velocity vector is (0, 0, 0),
+    external forces and torques vectors are also (0, 0, 0)"""
+
+    self.n_elems = 1
+    self.position = np.array([[0], [0], [0]])
+    self.director = np.array(
+        [[[1.0], [0.0], [0.0]], [[0.0], [0.0], [1.0]], [[0.0], [0.0], [1.0]]]
+    )
```

Comment:
shouldn't second and third rows be orthogonal vectors?

Reply:
You are right! Apologies for this typo.